src/components/Composer/index.js

```diff
@@ -161,6 +161,7 @@ function Composer({
     checkComposerVisibility,
     selection: selectionProp,
     isReportActionCompose,
+    isComposerFullSize,
```

Comment:
Missing prop-types and the default value for `isComposerFullSize`.

Reply:
Done